app/views/refinery/_menu.html.erb

```diff
@@ -11,12 +11,3 @@
     %>
   </div>
 <% end %>
-
-<script>
```

Comment:
So does this mean we have to remember to add those pages into the menu on staging and prod from within Refinery?

Reply:
yes!